accessibility/README.md

```diff
@@ -52,6 +52,13 @@ Automated checks can catch a lot of common issues before they reach production.
 
 ### Development
 
+- Ensure every focusable or interactive element has an [accessible name][]
+- Follow the [Cardinal Rules of Naming][]:
+  1. [Heed Warnings and Test Thoroughly][rule-1]
+  2. [Prefer Visible Text][rule-2]
```

Comment:
Would "Prefer Visible Text" also be a guideline worth including under this document's `Design` heading?

Reply:
I think that would make sense. What would you think of something a little more self-explanatory, like:

> Ensure interactive elements [have a text label][rule-2]

Reply:
Added!